### ISSUE TITLE
Wishlist for future versions/upgrades

### ISSUE THREAD

**Issue by SeanPoints** [kslong/kred#29]
1) Have the capability to use files from adjacent fields to fill tiles that lie on the edge. I think this may be possible in the SetupTile.py stage so the code looks through all tables for detectors that overlap the tile.

2) Maybe think about an alternate tile scheme.  Instead of 4x4, try 3x3 or 2x2.  This just involves changing MC_tiles.txt.

3) Improve continuum-subtraction (revisit hotpants and look for alternatives)

**Comment by kslong** [comment]
These are reasonable, and conform to what I was expecting. 

I, personally, am reluctant to revisit hotpants for image subtraction, because it is poorly documented and maintained.

Please add, here or as a separate issue, header keywords needed by NOIRLAB.

**Comment by SeanPoints** [comment]
I agree on avoiding hotpants, if possible.  I believe that we (you) resolved all of the header issues before our first upload to the Data Lab.  I will check with them again.

**Comment by kslong** [comment]
@SeanPoints  as for item 2 in your wish list.  This is already possible, but the correct approach is not to modify MC_tiles.txt.
Instead you should create a new table with the same format but whatever sizes and positions you want.  , because SetupTile.py already has a switch -xtab to select a different config file, which can be located either in the config directory or in the directory from which kred is being run.  It looks in the local directory first, so for experimentation you should create a new master run directory, and link the MEF directory only, and then put your modified config file there.  

There may be issues with the -all flag for some routines if you try to run in this mode, because there are a number of cases where the -all flag results, in the steer routines for the construction of the tile names, assuming there are 16 tiles per field.  I doubt this will stop the programs from running.

If this solution is not appropriate.  Please submit a single issue on this topic, explicitly indicating what the use case you want to have available.

**Comment by kslong** [comment]
Item 1 should be addressed with  5f2494dc442a2b906223d4bd45ae1b5313db2758

There is a new switch in SetupTile -use_all_data that looks at all of the data when identifying CCD images that contribute to a tile.
